astropy/wcs/tests/test_utils.py

```diff
@@ -1149,3 +1149,20 @@ def test_fit_wcs_from_points():
 
     assert dists.max() < 7e-6*u.deg
     assert np.std(dists) < 2.5e-6*u.deg
+
+
+def test_pixel_to_world_itrs():
+    """Regression test for https://github.com/astropy/astropy/pull/9609"""
+    wcs = WCS({'NAXIS': 2,
+               'CRPIX1': 168.0,
+               'CRPIX2': 84.5,
+               'CRVAL1': 0,
+               'CRVAL2': 0.0,
+               'CDELT1': -0.9675041009449499,
+               'CDELT2': 0.9646246244540423,
+               'CTYPE1': 'TLON-AIT',
+               'CTYPE2': 'TLAT-AIT',
+               'RADESYS': 'ITRS ',
+               'DATE-OBS': '2017-08-17T12:41:04.444'})
+    # Just check that this doesn't raise an error.
+    wcs.pixel_to_world(0, 0)
```

Comment:
Can you capture the output and feed it back to `world_to_pixel`` and check that the values round trip? This will also make sure we don't run into any spherical/cartesian issues with world_to_pixel.

Reply:
I'd like to request also two types of inputs - scalars, as in the current example and arrays.

Reply:
> Can you capture the output and feed it back to `world_to_pixel`` and check that the values round trip? This will also make sure we don't run into any spherical/cartesian issues with world_to_pixel.

I'm getting an exception because it's trying to download the IERS A bulletin. How do I disable this for the unit tests?

```
E           astropy.utils.exceptions.AstropyWarning: failed to download ftp://cddis.gsfc.nasa.gov/pub/products/iers/finals2000A.all and https://datacenter.iers.org/data/9/finals2000A.all, using local IERS-B: <urlopen error Unable to open any source! Exceptions were {'ftp://cddis.gsfc.nasa.gov/pub/products/iers/finals2000A.all': URLError("ftp error: OSError('An attempt was made to connect to the internet by a test that was not marked `remote_data`. The requested host was: 198.118.242.40')"), 'https://datacenter.iers.org/data/9/finals2000A.all': URLError(OSError('An attempt was made to connect to the internet by a test that was not marked `remote_data`. The requested host was: datacenter.iers.org'))}>
```

Reply:
I mean, for sure, I can use `@pytest.mark.remote_data`, but that seems a little overkill.

Reply:
> I'd like to request also two types of inputs - scalars, as in the current example and arrays.

Done.

Reply:
> Can you capture the output and feed it back to `world_to_pixel`` and check that the values round trip?

Done.